sycl/test-e2e/Graph/Explicit/empty_with_deps.cpp

```diff
@@ -0,0 +1,58 @@
+// REQUIRES: level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: %{run} %t.out
+
+// Tests the explicit API interface for adding empty nodes, and that
+// no_cycle_check is accepted as a command_graph construction property.
```

Comment:
Could you update this comment so it is different from Explicit/empty.cpp test comment